src/app/globals.css

```diff
@@ -16,9 +15,9 @@ body {
   height: 100vh;
   color: var(--theme-color-text);
   background: linear-gradient(
-    to bottom right,
-    var(--background-gradiant-dark),
-    var(--background-gradiant-light)
+          to bottom right,
```

Comment:
[Please Change]: The whitespace changes in these files should not be part of this PR (see general comment on git history).